Sources/Helpers/OAAppSettings.m

```diff
@@ -3902,6 +3903,7 @@ - (instancetype) init
         _oprUseDevUrl = [[[OACommonBoolean withKey:oprUseDevUrlKey defValue:NO] makeGlobal] makeShared];
         _offlineEditing = [[[OACommonBoolean withKey:offlineEditingKey defValue:YES] makeGlobal] makeShared];
         _osmUseDevUrl = [[[OACommonBoolean withKey:osmUseDevUrlKey defValue:NO] makeGlobal] makeShared];
+        _mapperLiveUpdatesExpireTime = [[OACommonLong withKey:mapperLiveUpdatesExpireTimeKey defValue:0] makeGlobal];
```

Comment:
If this settings is in the Android, probably you need to add it to the registeredPreferences with the correct key